scripts/bootstrap-py2-venv

```diff
@@ -0,0 +1,53 @@
+# This script is meant to be sourced by bash, not executed.
+# As such, don't use exit, use return.
+
+# This script is purely a python virtualenv bootstrap, not the entire development environment.
+# For the entire thing, you should still use make bootstrap.
+# Eventually, the goal would be to have an idempotent single script that does
+# everything make bootstrap does and more (system dependencies and so on).
+
+# This... doesn't quite work.
+# set -e
+# trap "echo 'bootstrap FAILED.' && trap - ERR && return" ERR
+
+# So just going to sprinkle returns everywhere.
+
+command -v pyenv >/dev/null || {
+    echo "You need to install pyenv. https://develop.sentry.dev/environment/#python"
```

Comment:
Possible future improvement: use something like this https://stackoverflow.com/a/3124750/90297